README.md

```diff
@@ -24,6 +24,13 @@ The native libraries are not packaged with any relevant jars.  See also note in
  * Ubuntu 18.04+: `apt-get install libblosc1`
  * conda: Installing `bioformats2raw` via conda (see below) will include `blosc` as a dependency.
 
+If using features that rely on OpenCV (see the [Downsampling type](#downsampling-type) section below), minimum supported versions are:
+
+ * Ubuntu 18.04
+ * RHEL 8
+ * Windows 10
+  - expect to see warnings as described in https://github.com/opencv/opencv/issues/20113; these can be ignored
```

Comment:
This renders at the same level as the previous bullet point. Maybe turn into parentheses?

Reply:
19829f1 should fix the bullet points, I just missed a single space. Can put the note about warnings on the same line in parentheses if you think that's clearer though?